core/core.md

```diff
@@ -10,7 +10,7 @@ In order to understand the Fora protocol, it is useful to define the entities in
 
 **Host**: As mentioned above, a host is a member that maintains the full state of the community. They will receive messages from other members of the community over a gossip network and update their own state, as well as gossip any updates to the rest of the network. They may or may not serve as an access point for Light Users. Hosts will record and propogate community interactions, and they must also listen to events on the blockchain in order to abide by any decisions made there. An honest Host will delete any banned content from their local copy of the community and ignore any banned members by refusing to accept and propogate their messages.
 
-**Admittors**: Communities may optionally elect a trusted member to the role of admittor. Admittors are responsible for admitting new members into the community. They may also assign voting power to members (if this is allowed by the community). They are appointed and replacable by governance.
+**Admittors**: Communities may optionally elect a trusted member to the role of admittor. Admittors are responsible for admitting new members into the community. They may also assign voting power, weighted against their own voting power, to members (if this is allowed by the community). They are appointed and replacable by governance.
```

Comment:
Does this mean that communities can impose a cap on the total amount of tokens an admittor is allowed to distribute (e.g. 1/3 of the total supply of voting tokens for the community to Admittor A?). Or does it mean an Admittor can distribute arbitrary amounts of their own voting tokens, but all those voting tokens collectively account for 1/3 of the community's vote?

Either solution has different implications.

In the former, the Admittor might be holding on to a large number of  voting tokens that do not get distributed. So there are voting tokens waiting for future members that don't get distributed right away. It also places a cap on the number of members a particular admittor could admit (and by extension a total cap); which all of governance must vote to increment.

In the latter, every time an Admittor assigns new voting power; all. the other members have their individual voting power diminished. Though I guess, this is also true in the capped case.

I definitely think the latter approach is better, which is what I think you're describing. Just writing this out to ensure we're thinking of the same thing and for future reviewers.

Reply:
The latter is what I had in mind. Diminished voting power makes sense to me and I actually think it would be a feature. There's no reason the earliest members of a collective should automatically sustain a higher amount of voting power simply for being let in earlier. I think the concept of adding a member means some members voting power must decrease assuming the new member receives a non zero voting power. 

Admittors can just factor in the diminished voting power by assigning the new member a smaller amount (assuming they want to maintain the status quo mostly)

This is something we can probably iterate on as applications of Fora become more apparent

Reply:
Yeah the latter is what I had in mind. It should be encouraged for multiple admittors to give voting powers to the same honest users. That combats the part of ones voting power being diminished over time as one admitttor continually grants voting power to more users

will update

Reply:
Updated, @AdityaSripal's description, "Admittor can distribute arbitrary amounts of their own voting tokens" illustrates this sufficiently